Automattic/jetpack
# My Jetpack: Add Scan interstitial, Product detail card, add-scan route and data

Adds Scan interstitial, Product detail card, add-scan route and data

#### Changes proposed in this Pull Request:

* Add Scan interstitial, Product detail card, add-scan route and data

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



No

#### Testing instructions:






* Build My Jetpack

```bash
> jetpack build packages/my-jetpack
```

* Go to My Jetpack dashbord
* Confirm you see the Scan Interstitial by clicking on the `Add scan` link

<img width="200" alt="image" src="https://user-images.githubusercontent.com/77539/153244738-3bdc00f2-d58e-4536-990a-8dea3c50f2e4.png">

<img width="800" alt="image" src="https://user-images.githubusercontent.com/77539/153244822-c7e6b814-0b9b-4aa2-86bf-1c5b9b929c8e.png">

**Storybook**

* Run storybook

```bash
cd ./jetpack/projects/js-packages/storybook
```

```bash
> pnpm install & pnpm storybook:dev
```

<img width="800" alt="image" src="https://user-images.githubusercontent.com/77539/153244897-9e1c5b17-4b37-48e1-a5bb-05d9dcfe9757.png">
